salt/modules/win_lgpo.py

```diff
@@ -4237,20 +4250,26 @@ def _write_regpol_data(data_to_write,
                     version_nums = (0, 1)
                 elif gpt_extension.lower() == 'gPCUserExtensionNames'.lower():
                     version_nums = (1, 0)
-                gpt_ini_data = "{0}{1}={2}\r\n{3}".format(
+                gpt_ini_data = '{0}{1}={2}\r\n{3}'.format(
                         gpt_ini_data[general_location.start():general_location.end()],
                         'Version',
-                        int("{0}{1}".format(six.text_type(version_nums[0]).zfill(4), six.text_type(version_nums[1]).zfill(4)), 16),
+                        int("{0}{1}".format(six.text_type(version_nums[0]).zfill(4),
+                                            six.text_type(version_nums[1]).zfill(4)),
+                            16),
                         gpt_ini_data[general_location.end():])
             if gpt_ini_data:
                 with salt.utils.files.fopen(gpt_ini_path, 'wb') as gpt_file:
```

Comment:
@twangboy should the file be written in ASCII mode as well, since it is not being read in binary mode?